app/views/miq_ae_customization/_dialog_resource.html.haml

```diff
@@ -1,6 +1,6 @@
 -# Parameters:
--#   tab     DialogTab object
--#   curr_pos sending it upto server so it can be used to reorder widgets
+  -# tab     DialogTab object
+  -# curr_pos sending it upto server so it can be used to reorder widgets
```

Comment:
@skateman these multiline comments could be rewritten by [this example](https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md#consecutivecomments)